src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -472,23 +478,72 @@ public DocumentConflict ResolveToLatest(List<DocumentConflict> conflicts)
 
             return latestDoc;
         }
-
-        private void ResolveAttachmentsConflicts(DocumentsOperationContext context, DocumentConflict resolved, bool resolvedToLatest)
+        
+        private bool ResolveAttachmentsConflicts(DocumentsOperationContext context, DocumentConflict resolved, bool resolvedToLatest)
         {
-            using var scope = Slice.External(context.Allocator, resolved.LowerId, out var lowerId);
-            var storageAttachmentsDetails = _database.DocumentsStorage.AttachmentsStorage.GetAttachmentDetailsForDocument(context, lowerId);
-            List<AttachmentName> resolvedAttachmentsMetadata = null;
+            using var scope = Slice.External(context.Allocator, resolved.LowerId, out var lowerIdSlice);
+            var storageAttachmentsDetails = _database.DocumentsStorage.AttachmentsStorage.GetAttachmentDetailsForDocument(context, resolved.LowerId);
+            var resolvedAttachmentsMetadata = AttachmentsStorage.GetAttachmentsFromDocumentMetadata(resolved.Doc).Select(attachment => JsonDeserializationClient.AttachmentName(attachment)).ToList();
```

Comment:
please put `_database.DocumentsStorage.AttachmentsStorage` into a local var, to reduce the length of lines here

Reply:
done